### PR TITLE
Communicators and groups

### PR DESCRIPTION
**Pull request by robertodr**
At the eleventh and a half hour...

### CODE REVIEW

File: content/communicators-groups.rst [hunk 1]
```diff
@@ -10,25 +10,68 @@ Communicators and groups
...
+     distinguish library and host messages.
+   - Learn the difference between :term:`intracommunicator` and
+     :term:`intercommunicator`.
+   - Learn how to create new communicators.
 
 
+.. typealong:: Something with intercommunicators
 
-.. todo::
+   sdfasdfoaj
```
Comment (wikfeldt) [inline content/communicators-groups.rst]: is this intentional?

**Review by wikfeldt — COMMENTED** [review]
looks like a good exercise! Faded examples 👍  😁

File: content/communicators-groups.rst (hunk 1 above)
Comment (robertodr) [reply content/communicators-groups.rst]: yes, just placeholder text to avoid a warning from Sphinx :smile_cat:

**Comment by robertodr** [comment]
Possibly, not my finest hour. This is an episode to work on for future editions.

File: content/code/comms-groups-create-typealong.c [hunk 2]
```diff
@@ -0,0 +1,36 @@
+#include <stdio.h>
```
Comment (mabraham) [inline content/code/comms-groups-create-typealong.c:1]: Should there be a -solution.c file for this?

**Comment by mabraham** [comment]
I would like to improve the style of the code used for the examples (constants not macros, declare variables at the point of use, not the top of the block C89-style), because students learn not only the intended content but the general qualities of examples, but that's not a job for today!

**Review by wikfeldt — APPROVED** [review]
since this is about to be taught it might be wise to merge it... On quick glance it looks good, improvements will have to wait until next iteration of this workshop

**Pull request merged**